docs/src/man/comparisons.md

```diff
@@ -235,6 +235,58 @@ The table below compares more advanced commands:
 | DataFrame as input        | `summarize(df, head(across(), 2))`                        | `combine(d -> first(d, 2), df)`                               |
 | DataFrame as output       | `summarize(df, tibble(value = c(min(x), max(x))))`        | `combine(:x => x -> (value = [minimum(x), maximum(x)],), df)` |
 
+## Comparison with the R package data.table
+
+The following table compares the main functions of DataFrames.jl with the R package data.table (version 1.14.1). To contrast data.table with dplyr, the focus has been put on transformations in place.
+
+|Operations              | data.table  | DataFrames.jl|  
```

Comment:
Can you add a space before and after bars for consistency with above? Could you also align bars to make the code easier to read?

Reply:
Done (I think). This is easier to read (if you have a wide screen!)